exec/plugin-commands-script-runners/src/dlx.ts

```diff
@@ -43,6 +45,10 @@ export function help (): string {
       {
         title: 'Options',
         list: [
+          {
+            description: 'Do not use or save the modules cache',
+            name: '--ignore-modules-cache',
```

Comment:
it is a bit long for a CLI option. Maybe just `--no-cache`?

Reply:
I fear it would confuse the users. Many would think it ignore all the cache (including the store).